dist/samples/store-locator/index.js

```diff
@@ -33,7 +46,7 @@ function initMap() {
           stores.push({ name, location: { lat, lng }, address: "" });
           const marker = new google.maps.Marker({ position: { lat, lng } });
           marker.addListener("click", () => {
-            update(new google.maps.LatLng({ lat, lng }));
+            seeDetail(new google.maps.LatLng({ lat, lng }));
```

Comment:
i would maybe keep the update and seeDetail separate. update drives the rendering of the list and other information. see detail can just change the camera

Reply:
done